docs/built_in_extensions.rst

```diff
@@ -127,7 +128,14 @@ You can control formatting of errors returned from your service by overriding :m
 
     {"message": "Argument `foo` is required.", "error": "INVALID_ARGUMENTS"}
 
+You can change the HTTP port and IP using the `WEB_SERVER_ADDRESS` config setting:
```

Comment:
Nice one! How to do this has come up on the forum a few times before... hopefully not so much anymore.